.config/nextest.toml

```diff
@@ -7,3 +7,13 @@ path = "junit.xml"
 filter = 'package(gear-authorship)'
 retries = 5
 threads-required = "num-test-threads"
+
+[[profile.ci.overrides]]
+filter = 'package(ethexe-cli)'
+retries = 5
+threads-required = "num-test-threads"
+
+[[profile.ci.overrides]]
+filter = 'package(ethexe-observer)'
+retries = 5
+threads-required = "num-test-threads"
```

Comment:
Do you think we need this any more ?

Reply:
well, this may make sense since we have anvil as child process, asynchronous tasks and main thread

Reply:
Ok